scripts/ci/precheck_pep8.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+# build packages
+. $(cd $(dirname $0); pwd)/artifacts.sh
```

Comment:
Do you need to run this just for flake8?

Reply:
Not needed. But I keep it here to further testing the build.